index.html

```diff
@@ -183,7 +251,267 @@ <h4 class="ring"><a href="#" class="over">Wolt</a></h4>
 </div>
 
 <br>
+<div class="Categories">
+<div class="hard-fork">
+<div >
+<h4><a href="#" class="categories-title">Hard Fork ></a></h4>
+</div>
+<div class="categories-white">
```

Comment:
Kindly replace `<div class="categories-white">` with `<article>` tag to improve the use of HTML semantics. Do this for the others